### DISCUSSION TITLE
Github does not respond to sending SMS CODE when I click "Send SMS Code"

### DISCUSSION THREAD

**Discussion by nabildzr** [community/community#137811] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![Screenshot (88)](https://github.com/user-attachments/assets/35400943-49bb-4f39-9c0a-dc0101789a17)

When I entered the name and password, Two-factor authentication with code via SMS appeared, I clicked "Send SMS", after that a pop up like this appeared on the screen

**Comment by davevad93** [comment]
Hi @nabildzr ,

Try contacting support by opening a ticket [here](https://support.github.com/contact/account), select `Two-factor authentication recovery` and explain your issue.

**Reply by nabildzr** [reply]
thank you for the reply, my account is back

**Reply by davevad93** [reply]
You're welcome @nabildzr , glad you got your account back! 👍

**Comment by AngotyFitia** [comment]
Hi @davevad93 I had the same problem but as for me, the account isn't recover. Can you help me please??
![Uploading git.png…]()
